command/pki_list_children_command.go

```diff
@@ -38,10 +38,17 @@ func (c *PKIListChildrenCommand) Help() string {
 Usage: vault pki list-intermediates PARENT [CHILD] [CHILD] [CHILD] ...
 PARENT is the certificate that might be the issuer that everything should be verified against.
 CHILD is a list of paths to certificates to be compared to the PARENT, or pki mounts to look for certificates on.  
+<<<<<<< HEAD
```

Comment:
Looks like you might've committed conflicts?

Reply:
oops. I've pushed something to fix this since undoing pushes gets me in trouble, but thank you very much for catching this.